toolchain/sem_ir/facet_type_info.cpp

```diff
@@ -158,6 +187,28 @@ auto FacetTypeInfo::Print(llvm::raw_ostream& out) const -> void {
     }
   }
 
+  if (!extend_constraints.empty()) {
```

Comment:
Maybe helpers might be good to reduce types? Granted the second field name changes -- don't know if that'd make you want to avoid the idea

Reply:
To be sure, I was assuming you'd pass the field as a template parameter. i.e.

template<typename SomeT, SomeFieldT>
auto Foo(...);

Foo(my_list, &Type::my_field);

To be sure, just wanted to be sure you've considered that kind of approach versus a lambda (presumably as a callback?).